src/components/OrderScreen.jsx

```diff
@@ -61,6 +62,10 @@ class OrderScreen extends Component {
     this.props.updateOrderDetails({ ...details, ...detailOverride });
   }
 
+  cancelForm() {
+    this.updateStoreFromForm({ stage: 'draft', dropoff: null });
```

Comment:
I'm going off of memory here, so feel free to tell me if I'm wrong.

Is the dropoff the only one we want to nullify?

Reply:
Great question. So basically saving the pickup location doesn't create the issue. And so the minimal solution is not saving the dropoff. Also the pickup is possibly logical to keep since what is canceled is the order - hence the requested dropoff is irrelevant - but the pickup location really remains the same (physically you're still there).

Reply:
I think the logic we want is that the pick up defaults to your current location if not set. So it makes sense to remove that too.

Reply:
Added cleanup for pickup location as well.
Re. setting the pickup to current location - I have created a different issue for this one - #76